CHANGELOG.md

```diff
@@ -19,9 +19,44 @@ Working tracking of changes, updated as work done prior to release.  Please revi
   - keys/salts per value kind (PII, item id, etc)
 
 
+## v0.4.9
+
+Breaking Changes: (at least, somewhat breaking)
```

Comment:
just an idea to improve readability is to split this by deployment kind, so customers in AWS don't need to read GCP stuff and viceversa. Like:

## v0.4.9
- Breaking changes (if any, for common stuff)
- Features (if any, for common stuff)
### Changes for AWS Deployments
- Breaking changes (if any)
- Features (if any)
### Changes for GCP Deployments
- Breaking changes (if any)
- Features (if any)

Reply:
well, it's a 2x2 matrix kinda - host platform X data source. So in this case breaking changes if EITHER deploying to GCP *or* using Google Workspace.